controllers/servicediscovery/servicediscovery_controller.go

```diff
@@ -724,7 +724,7 @@ func (r *Reconciler) ensureLightHouseAgent(instance *submarinerv1alpha1.ServiceD
 		return errors.Wrap(err, "error reconciling agent deployment")
 	}
 
-	err := metrics.Setup(instance.Namespace, instance, lightHouseAgent.GetLabels(), 8082, r.config.Client,
+	err := metrics.Setup("", instance.Namespace, instance, lightHouseAgent.GetLabels(), 8082, r.config.Client,
```

Comment:
```suggestion
	err := metrics.Setup(names.ServiceDiscoveryComponent, instance.Namespace, instance, lightHouseAgent.GetLabels(), 8082, r.config.Client,
```

Better specify it explicitly, makes code easier to follow